framework/OutStreams/OutStreamPlot.py

```diff
@@ -958,8 +958,6 @@ def addOutput(self):
             plt.subplot(self.gridSpace[x[0]:x[-1], y[0]:y[-1]])
           else:
             self.plt3D = plt.subplot(self.gridSpace[x[0]:x[-1], y[0]:y[-1]], projection = '3d')
-      elif self.dim == 3:
-        self.plt3D = plt.subplot(111, projection='3d')
```

Comment:
this was supposed to be in another PR, right? can you revert this?

Reply:
resolved.